src/popover/popover.directive.ts

```diff
@@ -86,6 +99,11 @@ export class PopoverDirective implements OnInit, OnDestroy {
         placement: this.placement,
         title: this.popoverTitle
       });
+
+    if (this.popoverCloseOnClickOutside) {
```

Comment:
Sure, but would be less efficient. Now listener is set only when this option is enabled and popover is shown, and removed after hiding.